Cargo.toml

```diff
@@ -36,10 +36,11 @@ itertools = "0.10.3"
 libc = "0.2.133"
 log = "0.4.16"
 nix = "0.26"
+malachite-bigint = { version = "0.1.0", git = "https://github.com/qingshi163/malachite-bigint.git" }
```

Comment:
👍 you got it 😄

Reply:
note that we can totally keep this under the RustPython github org to ease with maintainability. whatever you'd prefer.

Reply:
I don't mind to move it under RustPython org.

Reply:
totally, guess you can move it whenever you want. might be easier to finish up with it first in your local repo and then move.

Reply:
Either way will be ok up to @qingshi163's preference as the sole author.
Unlike sre-engine - though I don't know why this is out of RustPython repository, it is not necessarily a part of RustPython. But it will be good idea if you want to share maintenance burden.

Reply:
repr moved under RustPython org.